doc/user_guide/axes.rst

```diff
@@ -404,17 +417,55 @@ method:
 .. code-block:: python
 
     >>> axis.get_axis_dictionary()
-    {'name': <undefined>,
-    'units': <undefined>,
-    'navigate': <undefined>,
-    'size': 20,
-    'scale': 0.5,
-    'offset': 10.0}
+    {'_type': 'UniformDataAxis',
+     'name': <undefined>,
+     'units': <undefined>,
+     'navigate': <undefined>,
+     'size': 20,
+     'scale': 0.5,
+     'offset': 10.0}
 
 This dictionary can be used, for example, in the :ref:`initilization of a new
 signal<signal_initialization>`.
 
 
+Adding/Removing axes to/from a signal
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Usually, the axes are directly added to a signal during :ref:`signal 
+initialization<signal_initialization>`. However, you may wish to add/remove
+axes from the `AxesManager` of a signal.
+
+Note that there is currently no consistency check whether a signal object has
+the right number of axes of the right dimensions. Most functions will however
+fail if you pass a signal object where the axes do not match the data 
+dimensions and shape.
+
+You can **add a set of axes** to the `AxesManager` by passing either a list of
```

Comment:
Any reason to write "add a set of axes" in bold? I find it a bit too much, since there are very few things in the user guide which are in bold. Maybe italic would be enough if you want to highlight this part.
```
You can *add a set of axes* to the :py:class:`~.axes.AxesManager` by passing either a list of
axes dictionaries to :py:meth:`~.axes.AxesManager.create_axes`:
```

Reply:
ok